tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -755,33 +755,33 @@ engine renders in linear color space.
+| **int, float**       | hint_range(min, max[, step])                    | Restricted to values in a range (with min/max/step).                        |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | source_color                                    | Used as albedo color.                                                       |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | hint_normal                                     | Used as normalmap.                                                          |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | hint_default_white                              | As value or albedo color, default to opaque white.                          |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | hint_default_black                              | As value or albedo color, default to opaque black.                          |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | hint_default_transparent                        | As value or albedo color, default to transparent black.                     |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | hint_anisotropy                                 | As flowmap, default to right.                                               |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | hint_roughness[_r, _g, _b, _a, _normal, _gray]  | Used for roughness limiter on import (attempts reducing specular aliasing). |
+|                      |                                                 | ``_normal`` is a normal map that guides the roughness limiter,              |
+|                      |                                                 | with roughness increasing in areas that have high-frequency detail.         |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | filter[_nearest, _linear][_mipmap][_anisotropic]| Enabled specified texture filtering.                                        |
```

Comment:
The table width needs to be extended by 1 character, as the last character should *not* touch the pipe character (`|`).

Reply:
You are right, fixed